### PR TITLE
[INTERNAL] ESLint: Enforce max line length

### PR DESCRIPTION
**Pull request by RandomByte**

### CODE REVIEW

**Review by svbender — DISMISSED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -31,7 +31,16 @@ module.exports = {
 		"no-negated-condition": "off",
 		"require-jsdoc": "off",
 		"no-mixed-requires": "off",
-		"max-len": ["warn", 120],
+		"max-len": [
+			"error",
+			{
+				"code": 120,
+				"comments": 200,
+				"ignoreTrailingComments": true,
```
Comment (matz3) [inline .eslintrc.js:39]: Does this mean that you can put a comment with any length after a line of code?
I don't know whether this makes sense. Or maybe I don't understand the setting.

File: .eslintrc.js [hunk 2]
```diff
@@ -31,7 +31,16 @@ module.exports = {
 		"no-negated-condition": "off",
 		"require-jsdoc": "off",
 		"no-mixed-requires": "off",
-		"max-len": ["warn", 120],
+		"max-len": [
+			"error",
+			{
+				"code": 120,
+				"comments": 200,
```
Comment (matz3) [inline .eslintrc.js:38]: Why such a big difference between code and comments?

**Review by matz3 — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (RandomByte) [reply .eslintrc.js]: Removed

File: .eslintrc.js (hunk 2 above)
Comment (RandomByte) [reply .eslintrc.js]: Removed

**Review by matz3 — APPROVED** [review]

**Pull request merged**